### DISCUSSION TITLE
Encrypted HTML Files

### DISCUSSION THREAD

**Discussion by ds-brandao** [EvotecIT/PSWriteHTML#418] [Ideas]
It would be nice if we could encrypt the HTML files generated.

**Comment by PrzemyslawKlys** [comment]
What would be the use case? What are you trying to hide? You can already use minimize feature when you install PSParseHTML?

**Reply by ds-brandao** [reply]
I was thinking more for sensitive information safety. 

I am working on a project, and I like the project you guys are working on with PSWriteHTML, as well as the other repos available. However, the data I would like to use with it would be sensitive information and should be encrypted. Although, thinking about it, I could achieve this by other means, but I thought it would be a nice feature to add.

**Reply by PrzemyslawKlys** [reply]
You can't really hide anything with encryption without requiring a key to decrypt. The HTML has all the data, so once you open it up it will show it as is. You would need to add some javascript that would take all the existing html, encrypt it, and before showing it to user would prompt for password, but then the whole encryption mechanism would be in there - so no real security, as anyone could read it, and figure it out.
